### DISCUSSION TITLE
Discussion for feature other than CSS

### DISCUSSION THREAD

**Discussion by gilles-mnr** [Textualize/textual#370] [Q&A] [answered]
Hello @willmcgugan , i sent a message to you on Twitter for create another feature than css. Do i need to re-send it here ?

**Comment by willmcgugan** [comment, marked as answer]
Hi. You're welcome to contribute! But we're flat out on the CSS branch. It might be hard to contribute code until the API is more stable. Could you wait a month or two?

**Reply by gilles-mnr** [reply]
Of course. I'm going to study the code of the CSS. About the way to write the CSS, I think it would be better if this was the same of the true CSS. This would simplify the task to the Front-End developpers that would like to use this project (I think). I don't know if that is too late.

**Reply by gilles-mnr** [reply]
Sorry, i just saw that is patched in the last commit (text: __ on __ --> color & background)
